frontendservice/src/views/CustomerHome.vue

```diff
@@ -73,31 +82,30 @@ export default {
   components: {
     NavBarCustomer,
     cookieinfo,
-    cookieackno
+    cookieackno,
   },
 
   data() {
     return {
       isLoggedIn: false,
-      shops: []
+      shops: [],
     };
   },
 
   created() {
-    this.apifunction();
     var user = firebase.auth().currentUser;
     if (user) {
       this.isLoggedIn = true;
     }
-    setInterval(function() {
-      const self = this;
+    var self = this;
+    setInterval(function () {
       axios
-        .get("http://penguin.termina.linux.test:8085/shops")
-        .then(function(res) {
+        .get("http://penguin.termina.linux.test:8080/shops")
+        .then(function (res) {
           self.shops = res.data.shops;
         });
-    }, 2000 /*milliseconds */);
-  }
+    }, 500 /* milliseconds */);
```

Comment:
Let's keep this 2000 to have consistency across app.

Reply:
okay